apm-server.yml

```diff
@@ -1,54 +1,300 @@
-################### APM Server Configuration Example #########################
+######################## APM Server Configuration #############################
```

Comment:
In the Beats case we call it example because we already enable some modules by default and also change some of the default configs inside for a better getting started experience like overwriting number of shards / replicas. For APM perhaps the reference file could even be removed?